std/bigint.d

```diff
@@ -2312,12 +2312,13 @@ BigInt powmod(BigInt base, BigInt exponent, BigInt modulus) pure nothrow @safe
 
     while (exponent)
     {
-        if (exponent & 1)
+        if (exponent.data.peekUint(0) & 1)
         {
             result = (result * base) % modulus;
         }
 
-        base = ((base % modulus) * (base % modulus)) % modulus;
+        auto tmp = base % modulus;
+        base = (tmp * tmp) % modulus;
```

Comment:
Added a manual temporary value because `ldc2 -O3 --flto=full` (LDC version 1.22.0) doesn't perform common subexpression elimination here. Neither does `gdc -O4` (GDC version 10.2).